atomicapp/cli/main.py

```diff
@@ -53,9 +53,10 @@ def print_app_location(app_path):
 def cli_genanswers(args):
     try:
         argdict = args.__dict__
+        location = argdict['answers']
```

Comment:
Yeah, I was thinking that too, using `--destination` to coincide with how we do `fetch`. :)